README.md

```diff
@@ -45,7 +45,20 @@ The `ntripping` utility has the following usage:
             --url <url>             [default: na.skylark.swiftnav.com:2101/CRS]
 
 Different resources can be requested from different locations. By default, a San
-Francisco latitude, longitude, and height will be used.
+Francisco latitude, longitude, and height will be used. The data returned is in RTCM format.
+
+The `--url` must be formatted as such: `{username}:{passsword}@{area}.skylark.swiftnav.com:{port}/{mountpoint}`, 
```

Comment:
This is a skylark-ism and should be removed. This is a general NTRIP client, not a skylark client. Things like `{area}` are not applicable to a general NTRIP client. Further, the description of the resource is kind of redundant to the convention specified here: https://datatracker.ietf.org/doc/html/rfc1738#section-3.1.

Reply:
Generalized this